test/python/test_enable_optimizer.py

```diff
@@ -0,0 +1,87 @@
+import pandas as pd
+import duckdb
+
+# def test_optimizer_defaults_to_true(require):
```

Comment:
just to confirm it, the tests are commented out because we first need to merge these to make the tests pass right?

Should we add expect to fail for now or skip them (@pytest.mark.skip or xfail) , instead of commenting them out?

Reply:
Yes that's right! I've altered them for now with @pytest.mark.skip